libexec/mass-testing/revng-test

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+set -euo pipefail
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
+
+SECTIONS_JSON="$TEST_OUTPUT_DIR/sections.json"
+
+"$SCRIPT_DIR/readobj-json" "$TEST_INPUT" > "$SECTIONS_JSON"
+echo "text_size: $("$SCRIPT_DIR/text-size" "$SECTIONS_JSON")" >> "$TEST_OUTPUT_DIR/meta.yml"
```

Comment:
We usually prefer kebab-case

Reply:
Do you mean for the yaml keys? they tend to be annoying to handle in both python and typescript, I'd avoid them.

Reply:
Why? Don't we use `obj["text-size"]`?
No biggie anyway, in this case.